src/Modules/Manage/Dnn.PersonaBar.Users/Users.Web/src/_exportables/src/components/UserTable/UsersRoles/index.jsx

```diff
@@ -191,6 +198,7 @@ class UserRoles extends Component {
                     <GridCell columnSize={50}>
                         <span>
                             <Combobox suggest={false}
+                                ref={(dom) => {this.comboBoxDom = dom;}}
```

Comment:
usually we don't wanna make DOM operations out of React dataflow but I think this is fine for now. Later we can use our own common component rather than React Combobox